### ISSUE TITLE
Package for Firefox in Mozilla Debian repository does not configure gnome-www-browser or x-www-browser 

### ISSUE THREAD

**Issue by frodeseverin** [mozilla/kitsune#6180]
The package
`firefox`
from Mozilla Debian repository does not configure system-wide alternatives for `update-alternatives` for
`gnome-www-browser`
or
`x-www-browser`
Please see attached file for details.
[2024-08-11_Mozilla_Debian_repo.txt](https://github.com/user-attachments/files/16573252/2024-08-11_Mozilla_Debian_repo.txt)

I have a fresh install of Debian 12 Bookworm.
I have configured the Mozilla Debian repositories as per instructions on the [Mozilla website](https://support.mozilla.org/en-US/kb/install-firefox-linux#w_install-firefox-deb-package-for-debian-based-distributions).

I have modified apt pinning for the Mozilla repository by changing the line
`Pin-Priority: 1000
`
to
`Pin-Priority: 100`
and use 
`sudo aptitude -t mozilla`
to install specific packages from the Mozilla repository. This allows me to retain the firefox-esr package from the Debian repository. My current installed packages are
```
frodeseverin@Debian-virt-01:~$ apt-show-versions firefox
firefox:amd64/mozilla 129.0~build2 uptodate
frodeseverin@Debian-virt-01:~$ apt-show-versions firefox-esr
firefox-esr:amd64/bookworm-security 115.14.0esr-1~deb12u1 uptodate
frodeseverin@Debian-virt-01:~$ apt-show-versions firefox-devedition
firefox-devedition:amd64/mozilla 130.0b3~build1 uptodate
```

Please see attached file for deatils of my current situation.
[2024-08-11_Mozilla_Debian_repo.txt](https://github.com/user-attachments/files/16573252/2024-08-11_Mozilla_Debian_repo.txt)

Please add proper configuration for `update-alternatives` for`gnome-www-browser` and `x-www-browser`

**Comment by janbrasna** [comment]
Thanks for noticing @frodeseverin, it has already been reported — this is just not the right repository. Please follow https://bugzil.la/1905073 for more info (even when the bug doesn't sound relevant, the actual cause is exactly this.)

According to https://hg.mozilla.org/mozilla-central/rev/99d220318125 it should already be fixed as of 130.0a1 (nightly) from a week or two ago so you can test that yourself, and it will also probably find its way to ESR eventually.

The bug for this is https://bugzil.la/1863757

**Comment by frodeseverin** [comment]
Thank you for pointing me to the right repository.
I gather https//bugzil.la/ is for reporting bugs against Mozilla products.
And https://hg.mozilla.org/mozilla-central/ is repository for "Current Firefox and Gecko development."

And, yes, Firefox Nightly integrates into ùpdate-alternatives x-www-browser`.
I suppose it will make its way into the release build in due course.

I'll close this bug here, then.
